code/modules/mob/living/carbon/xenomorph/castes/shrike/shrike.dm

```diff
@@ -65,3 +65,12 @@
 		new_hud.add_hud_to(usr)
 	else
 		new_hud.remove_hud_from(usr)
+
+
+// ***************************************
+// *********** Icon
+// ***************************************
+/mob/living/carbon/xenomorph/shrike/proc/stop_screeching()
+	if(icon_state != "Shrike Screeching")
+		return
+	icon_state = "Shrike Running"
```

Comment:
```suggestion
	icon_state = "Shrike Running"

```
Try ending files with a new line.

Reply:
oh, i didn't know i was supposed to do that